### PR TITLE
💚 Add flake8 in CI

### PR DESCRIPTION
**Pull request by tkoyama010**
Let's add [flake8](https://pypi.org/project/flake8) in CI !

### CODE REVIEW

**Comment by tkoyama010** [comment]
Let's merge this PR after merging the #45 .

**Comment by GuillaumeFavelier** [comment]
@tkoyama010, now that #45 has been merged, feel free to update your PR :)

File: pyvistaqt/__init__.py [hunk 1]
```diff
@@ -1,3 +1 @@
 """PyVista package for 3D plotting and mesh analysis."""
```
Comment (tkoyama010) [inline pyvistaqt/__init__.py:1]:
```suggestion
"""PyVista package for 3D plotting and mesh analysis."""
from pyvistaqt._version import __version__
from pyvistaqt.plotting import BackgroundPlotter, MainWindow, QtInteractor

__all__ = ["__version__", "BackgroundPlotter", "MainWindow", "QtInteractor"]
```

**Comment by tkoyama010** [comment]
Now. It is ready for merge. Thank @GuillaumeFavelier for help. :rocket:

**Comment by tkoyama010** [comment]
Thanks @larsoner :smile:

**Pull request merged**